contributors/devel/sig-testing/testing.md

```diff
@@ -197,3 +197,61 @@ Please refer to [Integration Testing in Kubernetes](integration-tests.md).
 ## End-to-End tests
 
 Please refer to [End-to-End Testing in Kubernetes](e2e-tests.md).
+
+## Running your contribution in the Kubernetes CI
+Once you open a PR, [`prow`][prow-doc] runs pre-submit tests in CI.
+
+If you are not a [Kubernetes org member][member], another org member will need to run [`/ok-to-test`][ok-to-test] on your PR.
+
+Find out more about [other commands][prow-cmds] you can use to interact with prow through GitHub comments.
+
+### Troubleshooting a failure
+Click on `Details` and look at the [`gubernator`](gubernator.k8s.io/) output for the test.
+
+If the failure seems unrelated to the change you're submitting:
+- Is it a flake?
+  - Check if an issue has already been opened for that flake
+    - If not, open a new one (like [this example][new-issue-example]) and [label it `kind/flake`][kind/flake]
+  - Run [`/retest`][retest] on your PR to re-trigger the tests
+
+- Is it a failure that shouldn't be happening (in other words, is the test now wrong)
+  - Get in touch with the SIG
+    - preferably as a comment on your PR, by tagging the [team][k-teams] (for example a [reviewers team for the SIG][k-teams-review])
```

Comment:
anecdotally, does the use of these teams work as described here?

Reply:
Hm, interesting. @hoegaarden and I have had mixed experiences so far, i.e. it's not much overhead & it hasn't hurt to tag reviewers, it sometimes helped (but sometimes made no significant difference). Not sure how representative this is though; would you recommend a different suggestion here? 🤔

Reply:
no particular suggestion, I wish we had ways of measuring efficacy here